services/vision/client.go

```
@@ -242,6 +242,26 @@ func protoToObjects(pco []*commonpb.PointCloudObject) ([]*vision.Object, error)
 	return objects, nil
 }
 
+func (c *client) GetProperties(ctx context.Context, extra map[string]interface{}) *Properties {
+	ctx, span := trace.StartSpan(ctx, "service::vision::client::GetProperties")
+	defer span.End()
+
+	ext, err := protoutils.StructToStructPb(extra)
+	if err != nil {
+		return nil
```

You want to return the error